spec/lib/castle/client_spec.rb

```diff
@@ -54,18 +57,28 @@
 
   describe 'authenticate' do
     let(:request_response) { client.authenticate(event: '$login.succeeded', user_id: '1234') }
+    let(:response) do
```

Comment:
this should be named `request`, was reading a couple of times until I understood the test :)